src/app/frontend/download_data.py

```diff
@@ -46,3 +48,30 @@ def download_data(database: str, ids: List[int], search_term: str) -> None:
                 data=file_str,
                 file_name=f"{database}_{search_term}.fasta",
             )
+
+
+def download_seq_var_data(df: pd.DataFrame, species: str) -> None:
+    """Downloads sequence variability data as a FASTA file.
+
+    This function converts a DataFrame containing sequence variability data to
+    FASTA format and provides the option to
+    download the resulting FASTA file.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        A DataFrame containing sequence variability data.
+    species : str
+        The name of the species for which the data is being processed.
+    """
+    # Convert the DataFrame to FASTA format string
+    fasta_str = seqvar_df_to_fasta(df, species)
+
+    st.write("FASTA file ready for download!")
```

Comment:
```suggestion
    st.write("FASTA file is ready for download. Please click the button below to proceed.")
```

The more instructional the better for copy writing so it's clear for the user about the action that they should take 😄 I assume that the button is underneath this piece of text.